plugins/modules/docker_prune.py

```diff
@@ -109,33 +107,32 @@
 
 notes:
   - The module always returned C(changed=false) before community.docker 3.5.1.
-
 requirements:
   - "Docker API >= 1.25"
-'''
+"""
 
-EXAMPLES = '''
+EXAMPLES = r"""
 - name: Prune containers older than 24h
   community.docker.docker_prune:
     containers: true
     containers_filters:
-      # only consider containers created more than 24 hours ago
+    # only consider containers created more than 24 hours ago
       until: 24h
 
 - name: Prune containers with labels
   community.docker.docker_prune:
     containers: true
     containers_filters:
-      # Prune containers whose "foo" label has value "bar", and
-      # whose "bam" label has value "baz". If you only want to
-      # compare one label, you can provide it as a string instead
-      # of a list with one element.
+    # Prune containers whose "foo" label has value "bar", and
+    # whose "bam" label has value "baz". If you only want to
+    # compare one label, you can provide it as a string instead
+    # of a list with one element.
       label:
         - foo=bar
         - bam=baz
-      # Prune containers whose label "bar" does *not* have value
-      # "baz". If you want to avoid more than one label, you can
-      # provide a list of multiple label-value pairs.
+    # Prune containers whose label "bar" does *not* have value
+    # "baz". If you want to avoid more than one label, you can
+    # provide a list of multiple label-value pairs.
```

Comment:
```suggestion
      # Prune containers whose label "bar" does *not* have value
      # "baz". If you want to avoid more than one label, you can
      # provide a list of multiple label-value pairs.
```